cmd/config/init/error_handler.go

```diff
@@ -0,0 +1,7 @@
+package initconfig
+
+func handleError(err error) {
```

Comment:
maybe a better name would be `panicOnError` as there is no really handling of the error